packages/central-server/src/apiV2/requestPasswordReset.js

```diff
@@ -28,7 +28,7 @@ export const requestPasswordReset = async (req, res) => {
     user_id: user.id,
   });
 
-  const passwordResetUrl = `${process.env.TUPAIA_FRONT_END_URL}/reset-password?passwordResetToken={token}`;
+  const passwordResetUrl = `${process.env.TUPAIA_FRONT_END_URL}?passwordResetToken={token}#reset-password`;
   const resetUrl = passwordResetUrl.replace('{token}', token);
```

Comment:
Search params have to be after the hash so that the token isn't assumed by the browser to be part of the hash

Reply:
This will also be used in lesmis so will break if we change it. How about we add a redirect in the front end routes to redirect from /reset-password?passwordResetToken={token}` to ?passwordResetToken={token}#reset-password`?